ui/ui-frontend/projects/collect/src/app/collect/projects/create-project/create-project.component.ts

```diff
@@ -306,7 +307,7 @@ export class CreateProjectComponent implements OnInit, AfterViewChecked {
   }
 
   validateAndCreateProject() {
-    if (this.selectedWorkflow === Workflow.MANUAL) {
+    if (this.selectedWorkflow === Workflow.MANUAL || this.selectedWorkflow === this.Workflow.FLOW) {
```

Comment:
```suggestion
    if (this.selectedWorkflow === Workflow.MANUAL || selectedWorkflow === this.Workflow.FLOW) {
```

On peut avoir une autre valeur ? Je n'ai pas l'impression, donc on passe jamais dans le else ?

Reply:
La suggestion ne fonctionne pas, donc j'ai reformulé la syntaxe différemment.